app/Http/Controllers/Api/UnsubscribeController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Services\Manager;
+use Gladiator\Models\Competition;
+use Gladiator\Http\Requests\UnsubscribeRequest;
+
+class UnsubscribeController extends ApiController
+{
+    /**
+     * @var Gladiator\Services\Manager;
+     */
+    protected $manager;
+
+    public function unsubscribe(UnsubscribeRequest $request)
+    {
+        $credentials = $request->all();
+
+        $competition = Competition::find($credentials['competition_id']);
```

Comment:
We could sync on this, I was trying to find the best way to do an sql call, and I got the flow that it should be kept to the model, so I wasn't sure how to grab a Competition model to then do something like '$competition->unsubscribeUser'